docs/documentation.html

```diff
@@ -91,14 +91,15 @@
 
 .tileset_size
 .tile_coordinate
-
+<!--Cannot be invoked/inherited.-->
```

Comment:
?

Reply:
Originally from it being a class you'd think you can inherit the class like you can in ursina which was dead confusing to being with